orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroupLinearStageSupportSpec.groovy

```diff
@@ -82,7 +82,7 @@ class TargetServerGroupLinearStageSupportSpec extends Specification {
     def stage = new Stage(Execution.newPipeline("orca"), "test", ['region': 'should be overridden'])
 
     when:
-    def syntheticStages = supportStage.composeTargets(stage).groupBy { it.syntheticStageOwner }
+    def syntheticStages = supportStage.composeTargets(stage).findAll { !['acquireLock', 'releaseLock'].contains(it.type) }.groupBy { it.syntheticStageOwner }
```

Comment:
Couldn't we just add these to the assertion instead of excluding from selection?